code/modules/spells/spell_types/godhand.dm

```diff
@@ -110,11 +108,10 @@
 		to_chat(user, span_warning("You can't get the words out!"))
 		return
 	var/mob/living/M = target
-	if(M.anti_magic_check())
+	if(M.can_block_magic())		
```

Comment:
🥴

Reply:
Keep in mind I'm refactoring the antimagic component and not the entire magic spell files.  https://github.com/tgstation/tgstation/pull/64124/commits/c04476d86f43b35d892646f58594ba61332f6ff9